Tests/Command/FixMediaContextCommandTest.php

```diff
@@ -47,20 +45,29 @@ class FixMediaContextCommandTest extends CommandTest
     private $pool;
 
     /**
-     * @var \PHPUnit_Framework_MockObject_MockObject|ContextManagerInterface
+     * @var \PHPUnit_Framework_MockObject_MockObject|\Sonata\ClassificationBundle\Model\ContextManagerInterface
```

Comment:
@core23 @NeuralClone is completely right, plus there is no need to include a dependence if is not used in the code. 

Right now with the full qualified name as comment php can ran without error. If you add a dependence of a class that doesn't exist php will give you a error...